dataset_configs/portuguese/mcv/config.yaml

```
@@ -0,0 +1,90 @@
+documentation: |
+  MCV Portuguese
+  ##############
+
+  This config was originally designed for the
+  `Mozilla Common Voice (MCV) <https://commonvoice.mozilla.org/>`_ dataset
+  15.0 release, but should work for any subsequent releases as well.
+
+  It performs the following data processing.
+
+  1. Extracts and converts all data to the NeMo format.
+  2. Replaces certain non-supported characters, abbreviations and  punctuation marks with equivalent supported versions.
+  3. Drops any data that contains high/low character occurence.
+  4. Drops any data that contains symbols not in the supported alphabet.
+
+  **Required arguments**.
+
+  * **workspace_dir**: specify the workspace folder where all audio files will be stored.
+    You need to manually place the downloaded MCV Portuguese data inside
+    ``<workspace dir>/raw_data/`` subfolder.
+  * **data_split**: should be "train", "dev" or "test".
+
+  Note that you can customize any part of this config either directly or from command-line.
+  Here are some common customizations to consider:
+
+  **Output format**.
+
+  This config dumps the final manifest at ``${workspace_dir}/${data_split}_manifest.json``.
+  The output manifest contains the following fields:
+
+  * **audio_filepath (str)**: relative path to the audio files.
+  * **text (str)**: transcription, including punctuation ".,?" and capitalization.
+  * **duration (float)**: audio duration in seconds.
+
+
+
+processors_to_run: all
+workspace_dir: ???
+data_split: ???
+final_manifest: ???
+
+
+processors:
+  - _target_: sdp.processors.CreateInitialManifestMCV
+    raw_data_dir: ${workspace_dir}/raw_data
+    extract_archive_dir: ${workspace_dir}/row
```

```suggestion
    extract_archive_dir: ${workspace_dir}/raw
```
I think you meant this